setup.py

```diff
@@ -66,7 +66,7 @@ def get_installation_requirments(file_path):
 ]
 
 setup(name='atacworks',
-      version='0.2.0',
+      version='0.2.2',
```

Comment:
Is this PR supposed to be for master or dev 0.3.0? If for dev 0.3.0, then we should just update the version to be 0.3.0

Reply:
done.